wolfssl/wolfcrypt/settings.h

```diff
@@ -3583,6 +3583,11 @@ extern void uITRON4_free(void *p) ;
     #endif
 #endif
 
+#ifdef WOLFSSL_PYTHON
```

Comment:
Can you add a comment on why the WOLFSSL_PYTHON build needs the old OID sum?

Reply:
Comment added.